tee-worker/litentry/core/identity-verification/src/lib.rs

```diff
@@ -49,28 +46,11 @@ mod web3;
 mod error;
 use error::{Error, Result};
 
-pub fn verify(r: &IdentityVerificationRequest) -> Result<()> {
+pub fn verify(r: &IdentityLinkRequest) -> Result<()> {
```

Comment:
I think we should keep the old naming here, we are in the context of verification, not identity linking.

Reply:
Sounds good. I'll revert it - it's indeed a verification request.